cppcache/include/geode/DistributedSystem.hpp

```diff
@@ -108,16 +114,13 @@ class _GEODE_EXPORT DistributedSystem {
  private:
   std::string m_name;
   bool m_connected;
-  std::unique_ptr<statistics::StatisticsManager> m_statisticsManager;
-  std::unique_ptr<SystemProperties> m_sysProps;
+  std::shared_ptr<statistics::StatisticsManager> m_statisticsManager;
```

Comment:
Why are these suddenly `std::shared_ptr`?

Reply:
Short answer: `shared_ptr` is what would get it to build with our other changes.  Longer answer: when m_statisticsManager is a `unique_ptr` now, it breaks down somewhere in <memory> because some code path in std calls sizeof(StatisticsManager), which is unknown at compile time because the class is a forward reference.  Suppose this _could be_ solved with a custom move ctor, but is that the right thing to do?

The other looks like it was oversight, changed it back.